how-to/maintain/identify-abnormal-queries/identify-slow-queries.md

```diff
@@ -103,13 +103,63 @@ TiKV Coprocessor Task fields:
 
 > **Note:**
 >
 > Every time you query the `SLOW_QUERY` table, TiDB reads and parses the current slow query log.
 
-## Query example of SLOW_QUERY
+For TiDB 4.0, `SLOW_QUERY` supports querying the slow log of any period of time, including the rotated slow log file. You need to specify the `TIME` range to locate the slow log files that need to be parsed. If you don't specify the `TIME` range, TiDB only parses the current slow log file.
+
+* If you don't specify the time range, TiDB only parses the slow query data that TiDB is writing to the slow log file:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    select count(*),
+          min(time),
+          max(time)
+    from slow_query;
+    ```
+
+    ```
+    +----------+----------------------------+----------------------------+
+    | count(*) | min(time)                  | max(time)                  |
+    +----------+----------------------------+----------------------------+
+    | 122492   | 2020-03-11 23:35:20.908574 | 2020-03-25 19:16:38.229035 |
+    +----------+----------------------------+----------------------------+
+    ```
+
+* If you specify the time range, for example, from `2020-03-10 00:00:00` to `2020-03-11 00:00:00`, TiDB first locates the slow log files of the specified time range, and then parses the slow query information:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    select count(*),
+          min(time),
+          max(time)
+    from slow_query
+    where time > '2020-03-10 00:00:00'
+      and time < '2020-03-11 00:00:00';
+    ```
+
+    ```
+    +----------+----------------------------+----------------------------+
+    | count(*) | min(time)                  | max(time)                  |
+    +----------+----------------------------+----------------------------+
+    | 2618049  | 2020-03-10 00:00:00.427138 | 2020-03-10 23:00:22.716728 |
+    +----------+----------------------------+----------------------------+
+    ```
+
+> **Note:**
+>
+> If the slow log files of the specified time range are removed, or there is no slow query, the query returns NULL.
+
+TiDB 4.0 adds the [`CLUSTER_SLOW_QUERY`](/reference/system-databases/information-schema.md#cluster_slow_query-table) system table to query the slow query information of all TiDB nodes. The table schema of the `CLUSTER_SLOW_QUERY` table differs from that of the `SLOW_QUERY` table in that an `INSTANCE` column is added to `CLUSTER_SLOW_QUERY`. The `INSTANCE` column represents the TiDB node address of the row information on the slow query. You can use `CLUSTER_SLOW_QUERY` the way you do with [`SLOW_QUERY`](/reference/system-databases/information-schema.md#slow_query-table).
+
+When you query the `CLUSTER_SLOW_QUERY` table, TiDB pushes the computation and the judgment down to other nodes for execution, instead of retrieving all slow query information from other nodes to be executed by one TiDB.
```

Comment:
```suggestion
When you query the `CLUSTER_SLOW_QUERY` table, TiDB pushes the computation and the judgment down to other nodes, instead of retrieving all slow query information from other nodes and executing the operations on one TiDB node.
```
You have to say "one TiDB instance/node/etc" rather than "TiDB" alone.